sktree/tests/test_supervised_forest.py

```diff
@@ -21,6 +34,29 @@
     random_state=0,
 )
 
+# Larger regression sample used for testing feature importances
+X_large_reg, y_large_reg = make_regression(
+    n_samples=500,
+    n_features=10,
+    n_informative=3,
+    shuffle=False,
+    random_state=0,
+)
+
+FOREST_CLASSIFIERS = {
+    "ObliqueRandomForestClassifier": ObliqueRandomForestClassifier,
+    "PatchObliqueRandomForestClassifier": PatchObliqueRandomForestClassifier,
+}
+
+FOREST_REGRESSORS = {
+    "ObliqueRandomForestRegressor": ObliqueRandomForestRegressor,
+    "PatchObliqueRandomForestRegressor": PatchObliqueRandomForestRegressor,
+}
```

Comment:
These aren't used anywhere it looks like?

Reply:
I thought about using these to factor some of the repeated tests for classifier & regressor then I realized I defined wrong but I fixed it. Used in `test_check_importances_oblique`, `test_check_importances_patch`,`test_sklearn_compatible_estimator`